src/bttrx_control.h

```diff
@@ -63,14 +64,16 @@ class BTTRX_CONTROL {
 
 	ParameterType stringToParameterType(string);
 	string ParameterTypeToString(ParameterType);
+	ResultType handleSetCallsign(string);
 	ResultType handleSetADCGain(string);
 	ResultType handleSetDACGain(string);
 	ResultType handleSetPinCode(string);
 	ResultType handleSetPTTToggleEnabled(string);
 	ResultType handleSetPTTTimeout(string);
 	ResultType handleSetPTTHangTime(string);
 
+	string callsign_ = "";
```

Comment:
No need to store this locally as member, as we get this information directly from the Preferences object with BTTRX_CONTROL::get(...)

Reply:
Solved with https://github.com/phl0/firmware/commit/9bece9392f208d0897e485a552e37f3b87ffc408. (Will rebase to 1.1.0 dev branch as soon as all issues are done).